app/src/main/kotlin/cz/covid19cz/erouska/ui/sandbox/SandboxVM.kt

```diff
@@ -102,6 +106,25 @@ class SandboxVM(
         }
     }
 
+    fun scheduleDownloadKeyExport(context: Context) {
+        val constraints = Constraints.Builder()
+            .setRequiredNetworkType(NetworkType.CONNECTED)
+            .build()
+        val worker = PeriodicWorkRequestBuilder<DownloadKeysWorker>(
+            AppConfig.keyExportPeriod,
+            TimeUnit.HOURS
+        ).setConstraints(constraints)
+            .addTag(DownloadKeysWorker.TAG)
+            .build()
+
+        WorkManager.getInstance(context)
+            .enqueueUniquePeriodicWork(
+                DownloadKeysWorker.TAG,
+                ExistingPeriodicWorkPolicy.KEEP,
```

Comment:
Nemá tady spíš být REPLACE? KEEP funguje tak, že když je něco už naschedulované, nic to neudělá.

Reply:
Jo jo, máš pravdu, třeba při změně remote configu by se už nic nezměnilo